app/main/viewmanager.ts

```diff
@@ -67,6 +64,7 @@ class ViewManager {
 			return;
 		}
 		this.selectedIndex = index;
+		mainWindow.setBrowserView(null);
```

Comment:
What is the story behind this change?

Reply:
Unsetting the old view by doing this and then setting the new view ensures that there are no glitches while switching. Can be removed too, but i guess this is better.

Reply:
OK, I'd add a comment stating that here.

Reply:
Did you add anything earlier too? I did a force push recently.

Reply:
What do you mean by "Did you add anything"?